crates/sui-graphql-rpc/src/data/pg.rs

```diff
@@ -164,9 +153,19 @@ mod query_cost {
         };
 
         if cost > max_db_query_cost as f64 {
-            warn!(cost, max_db_query_cost, exceeds = true, "Estimated cost");
+            warn!(
```

Comment:
Made an earlier comment about this -- but you don't need to make this change -- this message is already annotated with a unique label (`query_cost`). because of the module it is in.